module/VuFind/src/VuFind/Db/Service/TagService.php

```diff
@@ -1015,13 +1016,13 @@ public function getForResource(
      * the returned list WILL NOT include tags attached to records that are not
      * saved in favorites lists.
      *
-     * @param string $userId     User ID to look up.
-     * @param string $resourceId Filter for tags tied to a specific resource (null
-     * for no filter).
-     * @param int    $listId     Filter for tags tied to a specific list (null for no
-     * filter).
-     * @param string $source     Filter for tags tied to a specific record source
-     * (null for no filter).
+     * @param string|User  $userId     User ID to look up.
```

Comment:
As noted in other comments above and below, we might want to double-check the types, parameter names and description wrapping in this comment.

Reply:
Please the ignore the resourceId parameter, I made a note of it and it will be modified when the corresponding calling methods are being migrated.